pkg/cmd/repo/license/view/view_test.go

```diff
@@ -0,0 +1,263 @@
+package view
+
+import (
+	"bytes"
+	"net/http"
+	"testing"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/cli/cli/v2/internal/config"
+	"github.com/cli/cli/v2/internal/gh"
+	"github.com/cli/cli/v2/pkg/cmdutil"
+	"github.com/cli/cli/v2/pkg/httpmock"
+	"github.com/cli/cli/v2/pkg/iostreams"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestNewCmdView(t *testing.T) {
```

Comment:
**How should this test SPDX ID?**

The examples below are primarily license keys.  Granted most SPDX IDs are just capitalized versions of the license keys but I haven't done an holistic confirmation 🤷

Reply:
I'm not sure if it makes sense to test for both the key and the SPDX ID since we would just hardcode the exact same API response in both cases. 

I don't think I could make one of these tests fail without the other test also failing, unless I screw up the HTTP stubs in the test.

What are your thoughts?

Reply:
Also, to hopefully illustrate my point more, the endpoint appears to be completely case insensitive. All these examples work:

```
gh repo license view MIt
gh repo license view MiT
gh repo license view miT
```

Reply:
You make a fair point as I wrestled over the same question knowing that license keys and SPDX IDs should be the same, so its fair to punt because it isn't significant.  Making sure to ask the question while we're here 👍